### DISCUSSION TITLE
how do i delete saved "restor windows from disk"?

### DISCUSSION THREAD

**Discussion by HappyPenguinzz** [kangyu-california/PersistentWindows#324] [Q&A]
I currently have 6 and want to delete them all since they are not restoring windows to proper sizes/locations and anytime when I try to manually save one (by holding ctrl and naming it) it doesn't show up as an option when I go to restore again. currently on version 5.54
the icon is also not stuck on red.

**Comment by HappyPenguinzz** [comment]
Log Name:      Application
Source:        Application
Date:          5/24/2024 4:45:20 AM
Event ID:      9990
Task Category: None
Level:         Information
Keywords:      Classic
User:          N/A
Computer:      Happy
Description:
The description for Event ID 9990 from source Application cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

PersistentWindows: Restore finished in pass 2 with 3 windows recovered for display setting Display_LocM2560xM228_Res2560x1440__Display_Loc0x0_Res1920x1080


The message resource is present but the message was not found in the message table

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application" />
    <EventID Qualifiers="0">9990</EventID>
    <Version>0</Version>
    <Level>4</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2024-05-24T10:45:20.2437576Z" />
    <EventRecordID>64625</EventRecordID>
    <Correlation />
    <Execution ProcessID="14936" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>Happy</Computer>
    <Security />
  </System>
  <EventData>
    <Data>PersistentWindows: Restore finished in pass 2 with 3 windows recovered for display setting Display_LocM2560xM228_Res2560x1440__Display_Loc0x0_Res1920x1080
</Data>
  </EventData>
</Event>

**Comment by kangyu-california** [comment]
download LiteDbStudio https://github.com/mbdavid/LiteDB.Studio/releases
exit PW
run LiteDbStudio, open (aka Connect) C:\Users\\[User]\AppData\Local\PersistentWindows\PersistentWindows.1.1.0.0.db
select the entries that you want to delete using right mouse button, select "Drop collection"
![image](https://github.com/kangyu-california/PersistentWindows/assets/59128756/6b5532d2-5aea-4aad-b6e5-0e1132d5d6ef)
click the Run button to commit the change
click Disconnect button to close the db file

rerun PW and you are all set

**Comment by HappyPenguinzz** [comment]
Thank you so much! literally fixed my problem with the list and with the windows not sizing correctly.
